documentation/src/components/component.ts

```diff
@@ -27,7 +27,7 @@ enum TabValue {
 }
 
 interface JsDocTagParsed {
-    tag: string;
+    tag?: string;
```

Comment:
I'm curious of why we're making the `tag` property optional? 
I also can't find where the property is being used 🤔 What does `tag` represent?

Reply:
I'm not sure what this was before...I've removed it.

Also, when we update to a slightly newer version of the analyzer output, it looks like we'll be able to start using the internal typings, but we're not quite there yet.